### ISSUE TITLE
TypeError: cheerio.fromURL is not a function

### ISSUE THREAD

**Issue by user4302** [cheeriojs/cheerio#3391]
i use the following code
```
import * as cheerio from 'cheerio';
const data = await cheerio.fromURL(https://www.youtube.com/);
```
and i get the following error
`TypeError: cheerio.fromURL is not a function`

**Comment by loganknecht** [comment]
I am experiencing this issue as well, except it occurs when I call `replaceWith`

**Comment by loganknecht** [comment]
Please disregard. The solution to this was.

```
const cheerio = require("cheerio");
// ...
const cheerio_file_dom = cheerio.load(current_file_contents);
// ...
const script_elements = cheerio_file_dom("script[src$=.js]");
script_elements.each((index, current_script_elements) => {
    const original_src_value = current_script_elements.attribs["src"];
    const new_value = `{{ require_js(get_asset_url(".${original_src_value}")) }}`;

    cheerio_file_dom(current_script_elements).replaceWith(new_value);
});
```

I think @user4302 may be incorrectly referencing the wrong object for the `.fromURL` command as well.

**Comment by user4302** [comment]
> Please disregard. The solution to this was.
> 
> ```js
> const cheerio = require("cheerio");
> // ...
> const cheerio_file_dom = cheerio.load(current_file_contents);
> // ...
> const script_elements = cheerio_file_dom("script[src$=.js]");
> script_elements.each((index, current_script_elements) => {
>     const original_src_value = current_script_elements.attribs["src"];
>     const new_value = `{{ require_js(get_asset_url(".${original_src_value}")) }}`;
> 
>     cheerio_file_dom(current_script_elements).replaceWith(new_value);
> });
> ```
> 
> I think @user4302 may be incorrectly referencing the wrong object for the `.fromURL` command as well.

i was following this guide in the docs
https://cheerio.js.org/docs/api#fromurl
i just saw that i used `data` instead of `$` when assigning the variable
if i used `$` would it have worked?

**Comment by allison-truhlar** [comment]
@user4302 I encountered this same issue with .fromURL. I think it is because this function is not yet available in the published npm package for Cheerio. The top of the website mentions "Cheerio's website is still a work-in-progress, and covers Cheerio's next release that isn't available yet." If you look in the Github Cheerio repository, you can find the function .from URL in cheerio/lib/batteries. However, in the npm package, this path does not exist.

Using this code achieved the same goal as using fromURL for me:
````
const response = await axios.get(url);
const html = response.data;

// Use Cheerio to parse the HTML
const $ = cheerio.load(html);

**Comment by sonyarianto** [comment]
Hmm I got into this trap as well hehe. The .fromURL() it's not there anymore :)

**Comment by TheNoobiCat** [comment]
I agree with @allison-truhlar, performing the get request yourself using something like Axios seems to be the way to go.

**Comment by fb55** [comment]
Fixed with the [1.0 release](https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0)!

**Comment by user4302** [comment]
> Fixed with the [1.0 release](https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0)!

i barely rememeber why i even needed this and where i used it :(

i shall relearn it xD
